core/src/main/scala/kafka/log/UnifiedLog.scala

```diff
@@ -520,6 +522,11 @@ class UnifiedLog(@volatile var logStartOffset: Long,
       localLog.updateRecoveryPoint(offset)
     }
   }
+  def updateHighestOffsetInRemoteStorage(offset: Long): Unit = {
+    if (!remoteLogEnabled())
+      warn(s"Unable to update the highest offset with remote index as: $offset, the existing value: $highestOffsetInRemoteStorage since remote storage is not enabled.")
```

Comment:
Could we make it more readable like the following?

Unable to update the highest offset in remote storage with offset $offset since remote storage is not enabled. The existing highest offset is $highestOffsetInRemoteStorage.